docs/user/integrations.rst

```diff
@@ -6,8 +6,8 @@
        (Perhaps reuse this: https://about.readthedocs.com/images/homepage.png)
 
 
-Building documentation on every Git commit
-==========================================
+Continuous Documentation Deployment
+===================================
```

Comment:
I'm a very big fan of this!

Even shortening it to "Continuous Documentation" can be an interesting way of talking about the workflow and the technical platform at the same time.